app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -22,6 +24,8 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import kotlin.collections.ArrayList
 
+const val SEARCH_SHARED_PREFS_KEY = "123"
```

Comment:
🍏 Лучше в качестве ключа использовать что-то более осмысленное и уникальное, чтобы не случилось пересечений. 
```suggestion
const val SEARCH_SHARED_PREFS_KEY = "SEARCH_SHARED_PREFS_KEY"
```